### PR TITLE
Disable removed services

### LINKED ISSUE #254: scionlab-config: stop services when scionlab-services.txt is received
When we receive the configuration from the coordinator, we must disable the previous services for `scionlab.target` and enable the new ones that come in the `scionlab-services.txt` file.
We also have to stop the previous services before we disable them: if we remove services from the list, they are not stopped (they no longer belong to `scionlab.target`).

### PR DESCRIPTION
**Pull request by FR4NK-W**
Fixes #254

### CODE REVIEW

File: scionlab/hostfiles/scionlab-config [hunk 1]
```diff
@@ -298,20 +298,26 @@ def install_scionlab_service(tmpdir):
     scionlab_services = os.path.join(tmpdir, 'scionlab-services.txt')
     if not os.path.exists(scionlab_services):
         _error_exit("Could not find the scionlab-services.txt file in %s", tmpdir)
+    with open(scionlab_services, 'r') as f:
+        services = [service.strip() for service in f.readlines()]
+
+    # 1. Disable services no longer in the service list
+    res = subprocess.run(['systemctl', 'show', 'scion-*@*.service', '--property=Id'], stdout=subprocess.PIPE)
```
Comment (juagargi) [inline scionlab/hostfiles/scionlab-config]: Can we instead list the dependencies of `scionlab.target`? Something like `systemctl list-dependencies --plain scionlab.target`

File: scionlab/hostfiles/scionlab-config [hunk 2]
```diff
@@ -298,20 +298,26 @@ def install_scionlab_service(tmpdir):
...
+    previous_services = [service.split("=")[-1].strip() for service in res.stdout.decode('utf-8').split("\n\n")]
+    to_disable = [service for service in previous_services if service not in services]
+    for srv in to_disable:
+        subprocess.run(['systemctl', 'disable', '--now', srv], check=True)
 
-    # 1. remove existing links to scionlab.target
+    # 2. remove existing links to scionlab.target
```
Comment (juagargi) [inline scionlab/hostfiles/scionlab-config]: the symlinks are already removed with the `systemctl disable` call.

File: scionlab/hostfiles/scionlab-config (hunk 1 above)
Comment (juagargi) [reply scionlab/hostfiles/scionlab-config]: There is a bug in Ubuntu 16.04 where the dependencies are shown twice. If we care about 16.04 we should remember that bug.

Comment (FR4NK-W) [reply scionlab/hostfiles/scionlab-config]: Taken care of.
Done

File: scionlab/hostfiles/scionlab-config (hunk 2 above)
Comment (FR4NK-W) [reply scionlab/hostfiles/scionlab-config]: Done

**Pull request merged**